src/github.com/getlantern/flashlight/client/config.go

```diff
@@ -13,12 +13,13 @@ var (
 
 // ClientConfig captures configuration information for a Client
 type ClientConfig struct {
-	MinQOS         int
-	DumpHeaders    bool // whether or not to dump headers of requests and responses
-	DeviceID       string
-	FrontedServers []*FrontedServerInfo
-	ChainedServers map[string]*ChainedServerInfo
-	MasqueradeSets map[string][]*fronted.Masquerade
+	MinQOS              int
+	DumpHeaders         bool // whether or not to dump headers of requests and responses
+	DeviceID            string
+	FrontedServers      []*FrontedServerInfo
+	ChainedServers      map[string]*ChainedServerInfo
+	MasqueradeSets      map[string][]*fronted.Masquerade
+	ProxiedCONNECTPorts []int
```

Comment:
I made these ports part of the configuration so that we can quickly change them if we decide to add some.